sdks/go/pkg/beam/core/runtime/exec/translate.go

```diff
@@ -457,7 +457,7 @@ func (b *builder) makeLink(from string, id linkID) (Node, error) {
 					}
 				}
 
-			case graph.Combine:
+			case graph.Combine, graph.CombineGlobally:
 				cn := &Combine{UID: b.idgen.New(), Out: out[0]}
 				cn.Fn, err = graph.AsCombineFn(fn)
 				if err != nil {
```

Comment:
We should print out, in the urn switch below, what urn we're not handling in the default case. `default: // For unlifted combines`
We shouldn't be missing any urns really, but a paranoid check can't hurt.

The [doc for lifted combines](https://docs.google.com/document/d/1-3mEs3Y7bIkJ0hmQ6SiHpVIFu5vbY6Zcpw-7tOMVg4U/edit#heading=h.i62jhvh3wkod) doesn't mention any different handling for global combines though, which is frustrating, and the proto only mentions we should understand the combine components, not that there's a "accumulate things locally" option that doesn't have an iterable value...